contracts/stellar-interchain-token/src/tests/test.rs

```diff
@@ -379,6 +381,22 @@ fn add_minter_fails_without_owner_auth() {
     assert_auth_err!(user, token.add_minter(&minter2));
 }
 
+#[test]
+#[should_panic(expected = "HostError: Error(Contract, #8)")] // MinterAlreadyExists
+fn add_minter_fails_minter_already_exists() {
+    let env = Env::default();
+
+    let minter = Address::generate(&env);
+
+    let (token, _) = setup_token(&env);
+
+    assert_auth!(token.owner(), token.add_minter(&minter));
+
+    goldie::assert!(fmt_last_emitted_event::<MinterAddedEvent>(&env));
+
+    token.mock_all_auths().add_minter(&minter);
```

Comment:
```suggestion
    token.mock_all_auths().add_minter(&minter);
```
remove goldie assert since the method being tested here is the failure when re-addition, which doesn't emit an event